ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/Wizard/ScanConfigWizardFooter.tsx

```diff
@@ -8,25 +8,29 @@ export type ScanConfigWizardFooterProps = {
     onSave: () => void;
     isSaving: boolean;
     proceedToNextStepIfValid: (nextFunction: () => void, stepId: string) => void;
+    allClustersAreUnhealthy: boolean;
 };
 
 function ScanConfigWizardFooter({
     wizardSteps,
     onSave,
     isSaving,
     proceedToNextStepIfValid,
+    allClustersAreUnhealthy,
 }: ScanConfigWizardFooterProps) {
     const { isModalOpen, openModal, closeModal } = useModal();
     const firstStepId = wizardSteps[0].id;
     const lastStepId = wizardSteps[wizardSteps.length - 1].id;
 
     function renderButtons(activeStepId, onNext, onBack) {
+        console.log('ScanConfigWizardFooter', allClustersAreUnhealthy, activeStepId, wizardSteps[1].id);
         return (
             <>
                 {activeStepId !== lastStepId ? (
                     <Button
                         variant="primary"
                         type="submit"
+                        isDisabled={allClustersAreUnhealthy && activeStepId === wizardSteps[1].id}
```

Comment:
@alwayshooin @mansursyed Did we also want to make sure this only occurs in the "create" flow and not the "edit" flow?